gems/quilt_rails/README.md

```diff
@@ -260,10 +264,12 @@ class GraphqlController < ApplicationController
 end
 ```
 
+🗒️ You will see `Can't verify CSRF token authenticity` in the rails server log when making a request to this end point. This is to be expected with `protect_from_forgery with: :null_session` to keep CSRF protection while ensuring the session is nullified when a token is not send in. You can also use `self.log_warning_on_csrf_failure = false` in the controller to suppress this error.
```

Comment:
If someone adds the `protect_from_forgery with: :null_session` line to their controller, they may expect that this will solve the issue and couldn't expect to see you can expect to see the `Can't verify CSRF token authenticity` error line in the logs. 

The phrase `even after adding` is letting them know that this is expected even after adding the `protect_from_forgery with: :null_session` line to their controller, and that it's not an issue with their code. You then give the extra code they can add to suppress this message. 

It's to do with letting them know this is expected behaviour and reassure them it's not a problem to go down a rabbit hole trying to solve - make sense?

Reply:
I see the original question :+1: